internal/repository/sync_test.go

```diff
@@ -215,8 +222,44 @@ func TestClone(t *testing.T) {
 		}
 		defer os.Chdir(currentDir) //nolint:errcheck
 
-		repo, err := Clone(context.Background(), remoteTmpDir+"//", "", "")
+		repo, err := Clone(context.Background(), remoteTmpDir+"//", "", "", nil)
+		assert.Nil(t, err)
+
+		head, err := repo.r.Head()
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.Equal(t, commitID, head.Hash())
+
+		localRSLRef, err := repo.r.Reference(plumbing.ReferenceName(rsl.Ref), true)
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.Equal(t, remoteRSLRef.Hash(), localRSLRef.Hash())
+
+		localPolicyRef, err := repo.r.Reference(plumbing.ReferenceName(policy.PolicyRef), true)
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.Equal(t, remotePolicyRef.Hash(), localPolicyRef.Hash())
```

Comment:
we should probably pull this out into a test helper...I think we've done this already, possibly in the gitinterface sync tests? we could likely copy that helper over.

Reply:
What part should be pulled out into a test helper?

Reply:
the repeated assert local, remote refs match check. see: https://github.com/gittuf/gittuf/blob/main/internal/repository/common_test.go#L13-L29

Reply:
Fixed this, I did not have to clone the func, since they are in the same package